### DISCUSSION TITLE
NOTICE: OpenSSL 3 error with MOOSE conda channel server

### DISCUSSION THREAD

**Discussion by cticenhour** [idaholab/moose#22826] [News]
EDIT: This issue seems to have been fixed. As usual - if this continues to occur, please comment here or in a new discussion post.

----

When following the [MOOSE Getting Started Instructions for conda](https://mooseframework.inl.gov/getting_started/installation/conda.html) there have been many reports of SSL errors during the environment creation step. Those encountering the error may see something like the following:

```
RuntimeError: Download error (35) SSL connect error [https://conda.software.inl.gov/public/noarch/repodata.json]
    error:0A000152:SSL routines::unsafe legacy renegotiation disabled
```

This error is due to a setting on the INL network. The core issue is being worked on internally, but in the meantime, a downgrade to OpenSSL version 1 can serve as a workaround. Perform the following in your conda installation:

```
conda activate base
conda config --remove channels https://conda.software.inl.gov/public
conda install openssl=1 
conda config --add channels https://conda.software.inl.gov/public
```

Then, the MOOSE environment can be created as normal:

```
conda create -n moose moose-libmesh moose-tools
conda activate moose
```

Please create a new discussions post if issues are still experienced after performing these steps. Thank you!

**Comment by milljm** [comment]
This looks to have been fixed on the INL servers.

**Reply by cticenhour** [reply]
I'm also not seeing this on the outside if openSSL is restored to version 3. Will update this post - thanks!
